sys/usb/usbus/hid/hid.c

```diff
@@ -107,9 +107,9 @@ static void _init(usbus_t *usbus, usbus_handler_t *handler)
     hid->hid_descr.arg = hid;
 
     /*
-    Configure Interface as USB_HID interface, choosing NONE for subclass and
-    protocol in order to represent a generic I/O device
-    */
+       Configure Interface as USB_HID interface, choosing NONE for subclass and
+       protocol in order to represent a generic I/O device
+     */
```

Comment:
Are these changes a result of uncrustifying? Can you split it into a separate commit? It's hard to follow the actual changes otherwise. Same for all other occurences.

Reply:
Sure thing :) Done.

Reply:
Thanks!